srv/apns/http_api/response.go

```diff
@@ -0,0 +1,8 @@
+package http_api
+
+// APNSErrorResponse is struct to represent JSON data returned by APNS HTTP API
+// if push request is not successful
+type APNSErrorResponse struct {
+	Reason    string
+	Timestamp string
```

Comment:
This should have been a number (unix timestamp), but apple's docs weren't very clear.

`...DeliveryPoint=Unknown Failed: Failed to send payload to APNS: json: cannot unmarshal number into Go struct field APNSErrorResponse.Timestamp of type string`